src/core/qlist.cc

```diff
@@ -355,6 +355,40 @@ void QList::Push(string_view value, Where where) {
   }
 }
 
+string QList::Pop(Where where) {
```

Comment:
Just to think: we can create our own string_view type (that can contain string_view or short string for 19 symbols) and return it without allocations

Reply:
You don't need to optimize by space, so it can be just a variant<string, string_view>

Reply:
we can not return string_view if we pop. in any case it's not urgent and can not be done easily during the transition.
there is no regression here.